src/style.css

```diff
@@ -2,0 +1,86 @@
+@font-face {
+  font-family: Avenir, Arial, sans-serif;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy, Arial, sans-serif;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+body {
+  margin: 0;
```

Comment:
It's a good practice to add a fallback font-family in case the main one doesn't work (from the checklist).